nservicebus/upgrades/5to6.md

```diff
@@ -402,6 +402,10 @@ snippet: ConvertEventToObservable
 
 ## Transaction configuration API
 
+`config.Transactions().DoNotWrapHandlersExecutionInATransactionScope()` has been removed since transaction scopes are no longer used by non DTC transports to do delayed dispatch. If running in DTC mode with MSMQ or SQLServer you should opt out from DTC Transactions using .DisableDistributedTransactions() since it would give you better performance for the same level of consistency.
```

Comment:
`If running in DTC mode with MSMQ or SQLServer you should opt out from DTC Transactions using .DisableDistributedTransactions() since it would give you better performance for the same level of consistency.`

Which same level? The level you had i V5 with `DoNotWrapHandlersExecutionInATransactionScope` or the same level you have in V6 when running in DTC mode with MSMQ or SQLServer?